### ISSUE TITLE
2020 Accessibility analysis: Common alt text length adjustment needed

### ISSUE THREAD

**Issue by obto** [HTTPArchive/almanac.httparchive.org#1506]
The common alt text length metric should also include stats if we EXCLUDE images with no alt tags at all. See https://docs.google.com/spreadsheets/d/1UjEBhq0TfYxUpdpq5IuxjeHB4yqhJq4NOKEd6Dwwrdk/edit#gid=152817998

**Comment by rviscomi** [comment]
@obto any progress on this?

**Comment by rviscomi** [comment]
📟 paging @obto. Is this blocking authors' progress?

**Comment by obto** [comment]
This wasn't needed in the end. We can close the issue
